public/about/index.html

```diff
@@ -43,38 +43,38 @@
         <img src="img/logo.png" alt="Rotational" class="h-14 w-auto sm:h-14">
     </a>
     <div class="topnav" id="myTopnav">
-        
-        
-            <a 
+
+
+            <a
             class=""
             href="/services/">
-            Services 
+            Services
             </a>
-        
-            <a 
+
+            <a
             class=""
             href="/opensource/">
```

Comment:
@masskoder can you please install the Trailing Spaces extension in your VSCode so that these extra spaces are automatically removed before you push to Github?

The extension we use is:
shardulm94.trailing-spaces